portfolio/src/main/webapp/comments.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
```

Comment:
Comments?

Reply:
Not resolved

Reply:
Is the title tag suppose to be title of the current page or the current website? I interpreted it as title for the entire website so that is why it is "My Portfolio" for all the html pages.

Reply:
nevermind it is suppose to be title of a page.